.github/workflows/gclientgen.yaml

```diff
@@ -0,0 +1,75 @@
+name: Client Generator
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-24.04
+    strategy:
+      matrix:
+        go-version: [ '1.23' ]
+
+    steps:
+    - uses: actions/checkout@v4
+    - name: Setup Go ${{ matrix.go-version }}
+      uses: actions/setup-go@v5
+      with:
+        go-version: ${{ matrix.go-version }}
+        cache-dependency-path: |
+          go.work.sum
+          gclientgen/go.sum
+    - name: Display Go version
+      run: go version
+    - run: go mod download
+    - run: go -C gclientgen test ./...
+
+  lint:
+    runs-on: ubuntu-24.04
+    strategy:
+      matrix:
+        go-version: [ '1.23' ]
+
+    steps:
+    - uses: actions/checkout@v4
+    - name: Setup Go ${{ matrix.go-version }}
+      uses: actions/setup-go@v5
+      with:
+        go-version: ${{ matrix.go-version }}
+        cache-dependency-path: |
+          go.work.sum
+          gclientgen/go.sum
+    - name: Display Go version
+      run: go version
+    - run: go -C gclientgen mod tidy
+    - run: go mod download
+    - run: go -C gclientgen fmt ./...
+    - run: go -C gclientgen vet ./...
+    - name: Run `staticcheck`
+      uses: dominikh/staticcheck-action@v1.3.0
+      with:
+          install-go: false
+          cache-key: ${{ matrix.go-version }}
+          working-directory: gclientgen
+    - name: Detect Changes
+      run: git diff --exit-code
+
+  vuln:
```

Comment:
I would also add a step to run https://pkg.go.dev/mvdan.cc/unparam

Reply:
Done.